ocs_ci/templates/app-pods/fedora_dc.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         image: fedora
         resources:
           limits:
-            memory: "500Mi"
+            memory: "800Mi"
```

Comment:
@sidhant-agrawal intentional change?

Reply:
Yes, it was intentional.
And I see this fix is already merged in #1217

Reply:
Rebased the wip branch